### PR TITLE
Fix KeyError when Telegram message has no text

### LINKED ISSUE #126: [Telegram] KeyError: 'text' when bot was added in a group
When the bot was added in a group, it receives a message like `Lucas Rangel Cezimbra invited bot` but when bottery receives this message it comes without text, so raises an exception as below.
```
Traceback (most recent call last):
  File "/home/lucas/telegram-bot/.venv/lib/python3.6/site-packages/bottery/platform/telegram/engine.py", line 92, in polling
    await asyncio.gather(*tasks)
  File "/home/lucas/telegram-bot/.venv/lib/python3.6/site-packages/bottery/platform/telegram/engine.py", line 153, in message_handler
    message = self.build_message(data)
  File "/home/lucas/telegram-bot/.venv/lib/python3.6/site-packages/bottery/platform/telegram/engine.py", line 131, in build_message
    return Message(
KeyError: 'text
```
https://github.com/rougeth/bottery/blob/master/bottery/platform/telegram/engine.py#L133

[Telegram documentation](https://core.telegram.org/bots/api#message) says that text is a optional field.

Is this a bug or am I doing something wrong?

If is a bug, how can we get around? We can use `message_data.get('text')` and set text to `None` when we do not have text? Or just return `None` instead of a message?

If it makes sense I can open a pull request.

### PR DESCRIPTION
**Pull request by Lrcezimbra**
Fix #126

### CODE REVIEW

**Comment by leportella** [comment]
Thanks @Lrcezimbra!!

File: tests/test_platform_telegram.py [hunk 1]
```diff
@@ -33,6 +33,33 @@ def message():
     )
 
 
+@pytest.fixture()
+def data_message():
```
Comment (rougeth) [inline tests/test_platform_telegram.py]: I believe `message_data` would be a better name for this fixture.

**Comment by rougeth** [comment]
@Lrcezimbra could you also move the `tests/test_platform_telegram.py` to `tests/telegram/test_engine.py`

**Comment by Lrcezimbra** [comment]
> I would change the personal name of the contributor to something more generic such as John Snow ou Jane Smith :)

Really, I will change this.

-----

> @Lrcezimbra could you also move the `tests/test_platform_telegram.py` to `tests/telegram/test_engine.py`

I tried to do this, but in the actual project structure it raises an exception because have 2 test modules with the same name:

```
import file mismatch:
imported module 'test_engine' has this __file__ attribute:
  /home/lucas/bottery/tests/platform/messenger/test_engine.py
which is not the same as the test file we want to collect:
  /home/lucas/bottery/tests/platform/telegram/test_engine.py
```

This is described in [pytest documentation](https://docs.pytest.org/en/latest/goodpractices.html#tests-outside-application-code):

> Note that using this scheme your test files must have unique names

Instead of create `__init__.py` files as suggested in pytest documentation I thought about rename to `tests/messenger/test_engine_messenger.py` and `tests/telegram/test_engine_telegram.py`. What do you think about this?

**Comment by rougeth** [comment]
I prefer the `__init__.py` files, I is not just one test file, so I don't think writing the platform name in each one of them is a good idea.

**Comment by Lrcezimbra** [comment]
I think this last commit solves all.

**Comment by rougeth** [comment]
@Lrcezimbra some tests are broken, are able to fix them? if not, I can check them for you.

**Comment by Lrcezimbra** [comment]
@rougeth I didn't understand why tests are broken. `from tests.utils import AsyncMock` is from Bottery, so it should come with local imports. Where am I wrong?

**Comment by rougeth** [comment]
`isort` settings wasn't considering `tests` as a first party module. I think I fixed it, can you update your branch?

**Comment by Lrcezimbra** [comment]
Test coverage was 0% :confused: 
I am searching why. Do you have any idea?

**Comment by rougeth** [comment]
Really strange, I'm going to take a look at this.

**Comment by Lrcezimbra** [comment]
Maybe this is what is causing the problem:

> Now pytest will load the modules as tests.foo.test_view and tests.bar.test_view, allowing you to have modules with the same name. But now this introduces a subtle problem: in order to load the test modules from the tests directory, pytest prepends the root of the repository to sys.path, which adds the side-effect that now mypkg is also importable. This is problematic if you are using a tool like tox to test your package in a virtual environment, because you want to test the installed version of your package, not the local code from the repository.
> In this situation, it is strongly suggested to use a src layout where application root package resides in a sub-directory of your root:

from: https://docs.pytest.org/en/latest/goodpractices.html#tests-outside-application-code

**Comment by rougeth** [comment]
@Lrcezimbra there were lots of updates here. Could you take a look at the `tests/` folter structure on master? We have sub folders just like I suggested and I didn't need to do nothing special to make it work. If you update your branch, I believe it's going to work. Thanks!!

**Comment by Lrcezimbra** [comment]
Closing this in favor of #134

**Pull request closed without merging**